.changeset/spotty-dingos-push.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+fix: login with third-party apps wasn't working without "Manage Oauth Apps"-permission
```

Comment:
please reword the changeset so it is not just the commit text, you couldve written something like:
"Fixed an issue where users were not able to login using OAuth without having the 'manage-oauth-apps' permissions" or something like that